python/pyspark/sql/session.py

```diff
@@ -212,13 +212,17 @@ def __init__(self, sparkContext, jsparkSession=None):
         self._sc = sparkContext
         self._jsc = self._sc._jsc
         self._jvm = self._sc._jvm
+
```

Comment:
really not a big deal but let's revert this newline to leave related changes only.